src/lib/select/select.ts

```diff
@@ -360,6 +360,9 @@ export class MatSelect extends _MatSelectMixinBase implements AfterContentInit,
     this._multiple = coerceBooleanProperty(value);
   }
 
+  /** Whether the user wants to disable the offset. */
```

Comment:
I'm not sure whether people looking at the documentation would know what offset this is referring to. What about calling it something like `disableOptionCentering`?

Reply:
I've made the changes you requested for the input:
- changed type to boolean
- changed name of the variable
- changed the if statement check

Reply:
Note that this comment will end up on the docs site. Something along the lines of "Whether to center the active option over the trigger" would be clearer.